recipe_scrapers/poppycooks.py

```diff
@@ -0,0 +1,64 @@
+from ._abstract import AbstractScraper
+from ._utils import normalize_string, get_yields
+
+
+class PoppyCooks(AbstractScraper):
+    @classmethod
+    def host(cls):
+        return "poppycooks.com"
+
+    def title(self):
+        return normalize_string(
+            self.soup.find("h1", class_="fl-post-title").get_text()
+        )
+
```

Comment:
We try to include Author where possible, it can be gathered from the author itemprop here.

```suggestion
        )

    def author(self):
        author_meta = self.soup.find("div", itemprop="author")
        name_meta = author_meta.find("meta", itemprop="name")
        return normalize_string(name_meta["content"])

```

Reply:
*will require a test case update as well